pkgs/games/runescape3/default.nix

```diff
@@ -0,0 +1,109 @@
+{ stdenv, lib, buildFHSUserEnv, dpkg, glibc, gcc-unwrapped, autoPatchelfHook, fetchurl, wrapGAppsHook
+  , gnome2, xorg
+  , libSM, libXxf86vm, libX11, glib, pango, cairo, gtk2-x11, zlib, openssl
+  , libpulseaudio
+  , SDL2, xorg_sys_opengl, libGL
+}:
+let
+
+  version = "2.2.9";
+  pname = "runescape-launcher";
+
+  src = fetchurl {
+    url = "https://content.runescape.com/downloads/ubuntu/pool/non-free/r/${pname}/${pname}_${version}_amd64.deb";
+    sha256 = "0r5v1pwh0aas31b1d3pkrc8lqmqz9b4fml2b4kxmg5xzp677h271";
+  };
+
+  runescape = stdenv.mkDerivation {
+    name = pname;
```

Comment:
Maybe it would be cleaner to move `pname` and `version` directly here.
```diff
-    name = pname;
+    pname = "runescape-launcher";
+    version = "2.2.9";
```

Doing this will result in name = pname + version.

Reply:
See https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming

Reply:
true, can do that. but then I also have to call it with `stdenv.mkDerivation rec {...}` instead of `stdenv.mrDerivation {...}`